sotodlib/coords/demod.py

```diff
@@ -180,4 +180,24 @@ def from_map(tod, signal_map, cuts=None, flip_gamma=True, wrap=False, modulated=
         if wrap:
             tod.wrap('signal', signal_sim, [(0, 'dets'), (1, 'samps')])
         return signal_sim
-    
\ No newline at end of file
+
+
+def rotate_demodQU(tod, update_focal_plane=True):
+    """
+    Apply detectors' polarization angle calibration to the HWP demodulated Q and U timestreams to
+    place all detectors' Q and U timestreams in a common telescope frame. This updates tod.demodQ
+    and tod.demodU, in place.
+
+    Args:
+        tod : an axisManager object
+        update_focal_plane (bool, optional): Whether to set focal_plane.gamma angles to zero,
+        consistent with new coordinate reference. Make this true for polarization mapmaking
+        using make_map.
```

Comment:
Indent second+third lines of the parameter description, for proper formatting by sphinx.